source/nvda.pyw

```diff
@@ -81,6 +82,26 @@ def stringToBool(string):
 	except ValidateError as e:
 		raise argparse.ArgumentTypeError(e.message)
 
+
+def stringToLang(value: str) -> str:
+	"""Perform basic case normalization for ease of use.
+	"""
+	if value.casefold() == "Windows".casefold():
+		return "Windows"
+	value = value.replace("-", "_")
+	lang = value.split("_")[0].lower()
+	dialect = value.split("_")[1].upper() if "_" in value else None
+	# Validating the size of the codes.
+	# Further validation would require L{languageHandler} to be initialized.
+	if len(lang) != 2 or (dialect and len(dialect) != 2):

```

Comment:
It would be great if this could check that the `lang` and `dialect` is in the list of supported languages. These errors could include the available languages.